docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst

```diff
@@ -7,24 +7,44 @@ Haemolytic disease and other neonatal jaundice
 Disease Description
 -------------------
 
-Case definition in GBD 2017
-+++++++++++++++++++++++++++
+`Hemolytic disease`_ and `neonatal jaundice`_ are two separate etiologies by which
```

Comment:
Is jaundice an etiology, or is it a symptom (i.e. yellowish skin that is a visible sign of hyperbilirubinemia) for which hemolytic disease is one etiology (and, e.g. G6PD is another) that is a precursor to developing extreme hyperbilirubinemia and kernicterus?

Reply:
Oops yes--that is correct. When GBD uses the label "haemolytic disease and other neonatal jaundice", they mean "a collection of etiologies which lead to extreme hyperbilirubinemia", and they define collection to include {Rh disease, preterm birth, G6PD, and other causes}.

The 'other' makes it sound like they mean all etiologies of EHB, or at least all such etiologies that fed into the data they were able to find.

Updating accordingly.